app/javascript/packs/application.js

```diff
@@ -17,3 +17,9 @@ import '../application'
 //
 // const images = require.context('../images', true)
 // const imagePath = (name) => images(name, true)
+//* require jquery3
+//* require bootstrap
+//* require popper
+//* require jquery_ujs
+
+Notification.requestPermission().then(function(result) {})
```

Comment:
Has this been manually tested? I don’t believe I was prompted to show notifications in Firefox.